### PR TITLE
MDFtoISO.MDFtoISO version 1.0

### PR DESCRIPTION
**Pull request by Masamune3210**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92779)

### CODE REVIEW

**Comment by Masamune3210** [comment]
Hash is correct, not sure what the bot is on about

File: manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MDFtoISO.MDFtoISO
+PackageVersion: 1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://www.softsea.com/download.php?id=832996351
+  InstallerSha256: 7087A6A80F326F6AC8D839BAA14029BD101DB5213F1FC2BBD3F984F6901B098B
```
Comment (stephengillie) [inline manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml:10]:
```suggestion
  InstallerSha256: 216D4BCEA4BBE226F0C610C59B911CF9351FEA34178A1F8EC04DAD99F4F1BDBC
```

File: manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MDFtoISO.MDFtoISO
+PackageVersion: 1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://www.softsea.com/download.php?id=832996351
+  InstallerSha256: 216D4BCEA4BBE226F0C610C59B911CF9351FEA34178A1F8EC04DAD99F4F1BDBC
```
Comment (stephengillie) [inline manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml:10]:
```suggestion
  InstallerSha256: 7087A6A80F326F6AC8D839BAA14029BD101DB5213F1FC2BBD3F984F6901B098B
```

Comment (stephengillie) [reply manifests/m/MDFtoISO/MDFtoISO/1.0/MDFtoISO.MDFtoISO.installer.yaml:10]: This is to revert to the original hash. The multiple download links on the page confused me and I selected the wrong package yesterday. I do not know what hash is being received by the validation pipeline, but I can manually validate this with the reverted hash.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**